release-notes/2020.md

```diff
@@ -351,12 +293,6 @@ The following updates were made in this release:
 
 * Places and Places Monitor extension released for Flutter. See current [Flutter versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#flutter) for more information.
 
-## July 10, 2020
-
-### iOS Audience 2.0.2
-
-* Fixed an issue where UUID was not returned in getSdkIdentities API response.
-
 ## July 09, 2020
 
 ### Flutter
```

Comment:
Do we keep Flutter and react native other platforms in this file?

Reply:
Yes, we need to remove react-native-acp* and other wrappers related release notes. It was easier/ faster for me to clean up the extension/ master release notes for the same experience cloud extension.

Reply:
Let's clean up the remaining with foundation extensions updates.